crates/bevy_render/src/renderer/render_device.rs

```diff
@@ -52,8 +52,8 @@ impl RenderDevice {
     ///
     /// no-op on the web, device is automatically polled.
```

Comment:
Can you describe what the return value means here?

Reply:
Sure thing! Added the description about the return value from the wgpu docs.